### DISCUSSION TITLE
Question about deploying locally via docker-compose

### DISCUSSION THREAD

**Discussion by ovizii** [docusealco/docuseal#71] [Q&A] [answered]
The sample docker-compose.yml file shows this volume:

>     volumes:
>       - .:/data

I am wondering if that means that the whole current working directory will be mapped? 
I tried changing that to 

>     volumes:
>       - './data:/data'

and creating a data folder which should be mapped. Seems to work, the folder gets used and the result is that data contains: docuseal/attachments and docuseal/.env - so I am wondering whether to map:

>     volumes:
>       - './data:/data/docuseal'

Instead?

I am also wondering why docuseal.env contains this line:
`DATABASE_URL= # keep empty to use sqlite or specify postgresql database URL`
as I am using POSTGRES and those variables are all set inside docker-compose.yml

**Comment by AlexBTurchyn** [comment, marked as answer]
@ovizii The `.:/data` mapping is used because it's the shortest possible mapping for the docker run command :)
That's true that the current directory is shared - users can just change it to ./other_directory:/data if they want to have some other folder mapped.

docuseal.env was created for those who don't use docker-compose but use docker run. It's ok to use docker compose env variables - in this case you can just ignore the docuseal.env file.

**Reply by ovizii** [reply]
Thank you so much for explaining.
